modules/jaeger-config-ingester.adoc

```diff
@@ -0,0 +1,92 @@
+////
+This REFERENCE module included in the following assemblies:
+-rhbjaeger-deploy.adoc
+////
+
+[id="jaeger-config-ingester_{context}"]
+= Jaeger Ingester configuration options
+
+Ingester is a service that reads from Kafka topic and writes to another storage backend (Elasticsearch).  If you are using the `allInOne` or `production` deployment strategies, you do not need to configure the Ingester service.
+
+
+.Jaeger default parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+|spec:
+  strategy: streaming
+  ingester:
+    options: {}
+|Configuration options that define the Ingester service.
+|
+|
+
+|ingester:
```

Comment:
Not sure what this row represents, as the topic is described in the next row aswell. So this row seems redundant?